collections/_posts/2025/04/05/2025-04-01-improving-mesherys-ui-e2e-tests.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Improving Meshery UI's E2E Tests
+author: Meshery Maintainers
+date: 2025-04-04
+categories: 
+  - meshery
+  - open-source
+  - playwright
+  - e2e-testing
+  - contributors
+featured-image: /assets/images/logos/meshery-logo-light-text-side.svg
+---
+
+Hey Meshery Community 👋
```

Comment:
```suggestion
Hey Meshery Contributors 👋
```

Reply:
Given that this post is not directed at users in the community, but contributors.